pyposeidon/utils/cast.py

```diff
@@ -55,7 +55,7 @@ def symlink_files(rpath: str, ppath: str, filenames: list[str]) -> None:
         dst = os.path.join(rpath, filename)
         if os.path.exists(src):
             os.makedirs(os.path.dirname(dst), exist_ok=True)
-            if os.path.exists(dst):
+            if os.path.lexists(dst):
```

Comment:
What's the point of `lexists` here? Is this to support copying EOS files?

Reply:
I don't remember why I've changed it. I was trying to fix the symlink method. It works with `os.path.exists` so I will revert.

Reply:
removed